core/src/main/java/com/linecorp/armeria/common/AbstractRequestContextBuilder.java

```diff
@@ -433,6 +437,13 @@ protected final String path() {
         return path;
     }
 
+    /**
+     * Returns the {@link UUID} of the request.
+     */
+    protected final UUID uuid() {
+        return uuid;
+    }
```

Comment:
- Could you add a public setter of the `uuid` as well?
- We could start with `null` `UUID` and then call `UUID.randomUUID()` only when a user did not call `builder.uuid(UUID)` (i.e. `this.uuid == null`).